### ISSUE TITLE
[BUG] Playwright extension (VS Code) does not automatically close browser and also does not run test in parallel 

### ISSUE THREAD

**Issue by nnson0310** [microsoft/playwright#20588]
**Context:**
- Playwright Version: 1.30.0
- Operating System: Windows
- Node.js version: v18.12.1
- Browser: Chrominum (Emulate Mobile Devices)

**Describe the bug**

Yesterday after I upgraded npm package of playwright from 1.29.0 to 1.30.0 and also upgrade to Chrome ver 109, I just realized that browser is not automatically closed after finishing running test with Playwright Extension of Vs Code. 

Otherwise, I have to use `page.close()` method to close the browser after running each testcases like this:

`
test.afterEach(async ({page}) => {
   page.close();
 });
`

Besides that, when I running all testcases in a spec.ts file, those testcases does not run in paralle although I config testcase to run in parallel. If running with `npx playwright test`, everything works fine. 
It seems this is a bug of Playwright Extension. Also note that, everything works well with Playwright 1.29.0

Below is my configuration for playwright (playwright.config.ts)
```
import type { PlaywrightTestConfig } from '@playwright/test';
import { devices } from '@playwright/test';

require('dotenv').config();

const config: PlaywrightTestConfig = {
  testDir: './e2e_tests',
  timeout: 30 * 1000,
  expect: {
    timeout: 15000
  },
  fullyParallel: true,
  forbidOnly: !!process.env.CI,
  retries: process.env.CI ? 2 : 0,
  workers: process.env.CI ? 2 : undefined,
  // workers: 5,
  reporter: 'html',
  use: {
    actionTimeout: 0,
    headless: true,
    trace: 'retain-on-failure',
    userAgent: process.env.USER_AGENT,
    contextOptions: {
      ignoreHTTPSErrors: true,
      acceptDownloads: true
    },
    video: 'retain-on-failure',
    screenshot: 'only-on-failure',
    extraHTTPHeaders: {
      'User-Agent': process.env.USER_AGENT
    }
  },

  projects: [
    {
      name: 'Mobile Chrome',
      use: {
        ...devices['Pixel 5'],
        headless: false
      },
      retries: 2
    },
  ],
  outputDir: 'test-results/',
};

export default config;
```

Does it is the intention of current version? Or it is a bug?

**Comment by dgozman** [comment]
@nnson0310  Do you have the ["Show Browser" checkbox](https://github.com/microsoft/playwright-vscode#show-browsers) turned on? I'd recommend to read the [getting started guide](https://playwright.dev/docs/getting-started-vscode) to familiarize yourself with all the features available.

**Comment by nnson0310** [comment]
> @nnson0310 Do you have the ["Show Browser" checkbox](https://github.com/microsoft/playwright-vscode#show-browsers) turned on? I'd recommend to read the [getting started guide](https://playwright.dev/docs/getting-started-vscode) to familiarize yourself with all the features available.

Of course I checked the checkbox "Show brower" but after finish running testcases, I think it should automatically close browser. With version 1.29, it does automatically close browser after finish running testcases. "Show browser" is just a feature which make browser visible, I think it does not related to "close browser after running"

**Comment by dgozman** [comment]
> "Show browser" is just a feature which make browser visible, I think it does not related to "close browser after running"

This is by design, "show browser" leaves it open so you can inspect the final state, pick a locator or generate some code after that. It is a debugging feature and I don't think we are going to change the behavior. You can use "close all browsers" item to close the browser if you'd like.

**Comment by nnson0310** [comment]
> > "Show browser" is just a feature which make browser visible, I think it does not related to "close browser after running"
> 
> This is by design, "show browser" leaves it open so you can inspect the final state, pick a locator or generate some code after that. It is a debugging feature and I don't think we are going to change the behavior. You can use "close all browsers" item to close the browser if you'd like.

Oh. I suppose there is no help if it is default behavior. Thank you. But about running testcases parallel in a .spec.ts file, although it shows "Running xyz tests using 4 workers", it only display only one browser instances. 

![2023-02-03 09_37_31-Greenshot](https://user-images.githubusercontent.com/100401156/216499648-2be484a5-bff8-4dbb-af41-ab1cf3ede267.png)

**Comment by dgozman** [comment]
> it shows "Running xyz tests using 4 workers", it only display only one browser instances.

We'll look into that.

**Comment by nnson0310** [comment]
> > it shows "Running xyz tests using 4 workers", it only display only one browser instances.
> 
> We'll look into that.

Thank you so much.

**Comment by dgozman** [comment]
Fixed in VSCode 1.75.1.
